### PR TITLE
trying to make a benchmark of it

### PR DESCRIPTION
**Pull request by JkLondon**
This benchmark gives very random and strange results... Maybe there is a way to modify it?

### CODE REVIEW

File: erigon-lib/kv/mdbx/kv_mdbx_test.go [hunk 1]
```diff
@@ -1092,3 +1107,34 @@ func TestDB_BatchTime(t *testing.T) {
...
+		return tx.Put(table, u64tob(uint64(1)), u64tob(uint64(1)))
+	})
+	if err != nil {
+		b.Fatal(err)
+	}
+
+	// Ensure data is correct.
+	if err := db.View(context.Background(), func(tx kv.Tx) error {
+		for i := 1; i <= b.N; i++ {
+			b.ResetTimer()
```
Comment (AskAlexSharov) [inline erigon-lib/kv/mdbx/kv_mdbx_test.go:1127]: Reset timer outside of loop - once is enough

File: erigon-lib/kv/mdbx/kv_mdbx_test.go [hunk 2]
```diff
@@ -1092,3 +1107,34 @@ func TestDB_BatchTime(t *testing.T) {
...
+			v, err := tx.GetOne(table, u64tob(uint64(1)))
```
Comment (AskAlexSharov) [inline erigon-lib/kv/mdbx/kv_mdbx_test.go:1128]: Encode and allocate key once before reset timer, outside of loop

File: erigon-lib/kv/mdbx/kv_mdbx_test.go (hunk 1 above)
Comment (JkLondon) [reply erigon-lib/kv/mdbx/kv_mdbx_test.go]: done

File: erigon-lib/kv/mdbx/kv_mdbx_test.go (hunk 2 above)
Comment (JkLondon) [reply erigon-lib/kv/mdbx/kv_mdbx_test.go]: done

**Review by AskAlexSharov — APPROVED** [review]

**Pull request merged**